src/sentry/api/endpoints/prompts_activity.py

```diff
@@ -0,0 +1,85 @@
+from __future__ import absolute_import
+
+import datetime
+import calendar
+from django.db import IntegrityError, transaction
+from django.http import HttpResponse
+from django.utils import timezone
+from rest_framework import serializers
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+
+from sentry.api.base import Endpoint
+from sentry.models import PromptsActivity
+
+PROMPTS = ['releases']
+
+
+class PromptsActivitySerializer(serializers.Serializer):
+    feature = serializers.CharField(required=True)
+
+    def validate_feature(self, attrs, source):
+        if attrs[source] not in PROMPTS:
+            raise serializers.ValidationError('Not a valid feature prompt')
+        return attrs
+
+
+class PromptsActivityEndpoint(Endpoint):
+    permission_classes = (IsAuthenticated, )
+
+    def get(self, request):
+        """ Return all prompts that are dismissed or in snoozed period"""
+        prompts = PromptsActivity.objects.filter(
+            organization_id=request.GET['organization_id'],
```

Comment:
To make this generic we should think about how to handle features at different levels: user, org and project. In our validator we wanna check that user and feature are always present. Then, depending on the feature level we ensure that org and/or project are present, and use those as filters in the query. The return value could be a dict of the form org -> project -> feature data (and in cases where the project or org aren't specified, that level of the dict is missing from the return value).